modules/ROOT/pages/websocket/websocket-connector.adoc

```diff
@@ -0,0 +1,97 @@
+
+== Compatibility
+
+WebSocket Connector is compatible with:
+
+|===
+|Application/Service|Version
+|Mule Runtime|4.2.2 or higher
+|Java |1.8.0_x
+|Anypoint Studio|7.4 or higher
+|===
+
+== Prerequisites
+
+Before creating an app that uses WebSocket Connector, you must have access to the WebSocket target resource and Anypoint Platform. You must also understand how to create a Mule app.
+
+== Triggers [[triggers]]
+
+Triggers initiate the execution of a flow upon receiving a message or event notification. You can specify the following triggers for WebSocket-related events:
+
+* `inbound-listenerPort` exposes an endpoint to which external systems can connect and initiates the associated app flow when the endpoint receives a message.
+* `on-inbound-connection` initiates an app flow when a new connection to the Inbound Listener is established.
+* `outbound-listener` exposes an outbound endpoint and initiates the associated app flow when the endpoint receives a message from an external system.
+* `on_connection-closed` initiates an app flow when an inbound or outbound WebSocket is closed.
+
+== Operations [[operations]]
+
+WebSocket Connector supports the following operations:
+
+* `open-outbound-socket` connects to an external system and creates an outbound WebSocket.
+* `send` sends a message through a WebSocket.
+* `broadcast` sends a message to a group of other WebSockets.
+* `subscribe-groups` subscribes a WebSocket to one or more groups.
++
+The app can invoke this operation on the same WebSocket multiple times. Each time the app specifies a new group, it is added to the list of target groups for the broadcast.
++
+* `unsubscribe-groups` unsubscribes a WebSocket from one or more groups.
++
+The app can invoke this operation on the same WebSocket multiple times. Each time an app invokes this operation for a group, the specified group is removed from the list of target groups for the broadcast.
++
+* `close-socket` closes a WebSocket connection.
+
+== Security and Authentication
+
+WebSocket Connector supports the WS (Web Sockets) and WSS (Web Sockets over SSL/TLS) URI schemes for outbound endpoints:
+
+*  WS identifies a WebSocket server and resource name in a URI, but does not use SSL for authentication or TLS to encrypt the communication. To use WS for an outbound endpoint, specify `ws` in the URI:
++
+`ws://localhost:80`
++
+* WSS identifies a WebSocket server and resource name in a URI. In addition, it uses SSL for authentication and TLS to encrypt the communication. To use WSS for an outbound endpoint, specify `wss` in the URI:
++
+`wss://localhost:433`.
+
+For an inbound endpoint, TLS and other HTTP configurations are inherited from the configuration of the referenced HTTP Listener.
+
+For detailed about configuring TLS for a WebSocket, see <<websocket/websocket-connector-configuration.adoc#tls-configuration,TLS Configuration>>.
+
+== See Also
```

Comment:
This see also page is missing in all the other articles. Therefore, there's no navigation experience for a customer reading this documentation. What we had discussed is that all pages should have links to whatever makes sense to read next, and that order needs to be defined on content